test/e2e/crd_e2e_test.go

```diff
@@ -252,7 +252,7 @@ var _ = Describe("CRD Versions", func() {
 		// Check the error on the installplan - should be related to data loss and the CRD upgrade missing a stored version
 		Eventually(func() (*operatorsv1alpha1.InstallPlan, error) {
 			return crc.OperatorsV1alpha1().InstallPlans(testNamespace).Get(context.TODO(), s.Status.InstallPlanRef.Name, metav1.GetOptions{})
-		}).Should(And(
+		}, 90*time.Second).Should(And(   // InstallPlan timeout is 60 seconds.  This must be longer than that to get the "InstallPlanPhaseFailed" status
```

Comment:
I _think_ this is reasonable to me but I haven't dived too deep whether we potentially sweep controller issues under-the-rug over time by simply bumping the timeout by 50%. The only other small comment I had (and this is a nit more than anything): does it make sense to make the `90*time.Second` a constant variable and we can add the comment you added here above that variable, to help improve readability for this test case?

Reply:
Thanks for review.  I'll make a constant.  The message in the state is checked in addition to the failure state here.  So I think it's safe increasing the timeout.  WDYT?

Reply:
Okay I think I can buy that explanation and we should be fine to simply bump the timeout.

Reply:
Maybe we decrease the timeout to 75s though? Thoughts?

Reply:
Yes. that' should be enough.  I'll change it.  Thanks!

Reply:
The fact that this was timing out reeks of an underlying controller issue.